MANIFEST.in

```diff
@@ -0,0 +1,17 @@
+include *.txt
+include CODEOWNERS
+include Makefile
+include pylintrc
+include tox.ini
+recursive-include dataprofiler *.avro
+recursive-include dataprofiler *.csv
+recursive-include dataprofiler *.html
+recursive-include dataprofiler *.json
+recursive-include dataprofiler *.md
+recursive-include dataprofiler *.parq
+recursive-include dataprofiler *.parquet
+recursive-include dataprofiler *.py
+recursive-include dataprofiler *.txt
+recursive-include examples *.ipynb
+recursive-include examples *.md
+recursive-include examples *.png
```

Comment:
concerned these are included in the installation... they shouldn't be

Reply:
that is `ipynb`, `md`, and `png`

Reply:
if I take them out I receive 
```
missing from sdist:
  examples/DL-Flowchart.png
  examples/add_new_model_to_data_labeler.ipynb
  examples/column_name_labeler.ipynb
  examples/data_profiler_demo.ipynb
  examples/data_readers.ipynb
  examples/graph_data_demo.ipynb
  examples/great_expecations/README.md
  examples/great_expecations/ge_expect_column_value_confidence_for_data_label.ipynb
  examples/great_expecations/ge_expect_column_values_vs_profile_min.ipynb
  examples/great_expecations/ge_expect_profile_numeric_columns_diff_in_range.ipynb
  examples/great_expecations/ge_expect_profile_numeric_columns_percent_diff_in_range.ipynb
  examples/intro_data_profiler.ipynb
  examples/labeler.ipynb
  examples/merge_profile_list.ipynb
  examples/popmon_dp_loader_example.ipynb
  examples/regex_labeler_from_scratch/DataLabeler_from_scratch.ipynb
  examples/structured_profilers.ipynb
  examples/unstructured_profilers.ipynb
```
and `check-manifest` recommends adding those 3 lines (15 - 17)